packages/schemas/src/types/log/interaction.ts

```diff
@@ -74,7 +75,7 @@ export enum Action {
  */
 export type LogKey =
   | `${Prefix}.${Action.Create | Action.End}`
-  | `${Prefix}.${InteractionEvent}.${Action.Update | Action.Submit}`
+  | `${Prefix}.${InteractionEvent}.${Action.Create | Action.Update | Action.Submit}`
```

Comment:
We may need to update the label of new keys in Console.

Reply:
Issue created. By default the log name will be used as the label display in the console.